torchaudio/models/wav2vec2/model_utils.py

```diff
@@ -0,0 +1,131 @@
+# Originally from fairseq:
+# https://github.com/pytorch/fairseq/blob/0dfd6b624081fc4e1c72fc74ae0cd2de199c334c/fairseq/data/data_utils.py#L393-L517
+from typing import Optional, Tuple
+import numpy as np
```

Comment:
`torchaudio` does not list NumPy as dependency so this needs a workaround. The best solution is to replace the use of NumPy with equivalent PyTorch.